jetty-core/jetty-server/src/main/config/modules/threadpool-virtual.mod

```diff
@@ -37,6 +37,10 @@ etc/jetty-threadpool-virtual.xml
 ## Virtual threads name prefix.
 #jetty.threadPool.virtual.namePrefix=qtp<hashCode>-virtual-
 
-## Whether virtual threads inherits the values of inheritable thread locals.
-#jetty.threadPool.virtual.inheritInheritableThreadLocals=true
+## Max number of concurrent virtual threads.
+#jetty.threadPool.virtual.maxThreads=200
```

Comment:
```suggestion
#jetty.threadPool.virtual.maxThreads=20000
```

Reply:
Why is this `jetty.threadPool.virtual.maxThreads` here and `jetty.threadPool.maxThreads` in `threadpool-all-virtual.mod`